benchmark/collective/resnet/utils/reader_cv2.py

```diff
@@ -32,7 +32,8 @@ def _reader_creator(settings,
                     color_jitter=False,
                     rotate=False,
                     data_dir=DATA_DIR,
-                    pass_id_as_seed=0):
+                    pass_id_as_seed=0,
+                    threads=4):
```

Comment:
Both THREAD and BUF_SIZE can be arguments and then remove the line 25 and 26.

Reply:
Done.